collect_app/src/test/java/org/odk/collect/android/widgets/DrawWidgetTest.java

```diff
@@ -19,14 +19,15 @@
 import org.odk.collect.android.activities.DrawActivity;
```

Comment:
createWidget() passes an anonymous subclass of FakeQuestionMediaManager to the constructor. 
Adds field for current file - required by fake QMM. Default is null,  will be initialized to real file only by Current test.
New field provides generalised prompt text for all tests.
Lightly polishes identifiers.